iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/digitaltwin/DigitalTwinClientAndroidRunner.java

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.android.digitaltwin;
+
+import com.microsoft.azure.sdk.iot.android.helper.TestGroup1;
```

Comment:
TestGroup1?

Reply:
We split the android tests into several test groups so that they can run in parallel. You can't run android tests in parallel on a single device